scripts/index.js

```diff
@@ -1,14 +1,26 @@
 
 // Implement the following functions to fulfill the tests!
-function getLocationName() {}
-function getLocationCountry() {}
-function getLocationLatitude() {}
-function getLocationLongitude() {}
-function getDescription() {}
+function getLocationName(data) {
+    return data.name;
+}
+function getLocationCountry(data) {
+    return data.sys.country;
+}
+function getLocationLatitude(data) {
+    return data.coord.lat;
+
+}
+function getLocationLongitude(data) {
+    return data.coord.lon;
+}
+function getDescription(data) {
+    return weather.description;
```

Comment:
Not sure if this one makes the test pass? In any case, it's always a good idea for your functions to do their work only based on arguments, when possible.